### PR TITLE
update global nixpkgs archive

### PR DESCRIPTION
**Pull request by coffee-cup**

### CODE REVIEW

File: src/nixpacks/nix/mod.rs [hunk 1]
```diff
@@ -8,7 +8,7 @@ pub mod pkg;
 // This line is automatically updated.
 // Last Modified: 2023-01-02 17:04:24 UTC+0000
```
Comment (Milo123459) [inline src/nixpacks/nix/mod.rs:9]: should this maybe be updated?

File: src/nixpacks/nix/mod.rs [hunk 2]
```diff
@@ -8,7 +8,7 @@ pub mod pkg;
...
 // https://github.com/NixOS/nixpkgs/commit/293a28df6d7ff3dec1e61e37cc4ee6e6c0fb0847
```
Comment (Milo123459) [inline src/nixpacks/nix/mod.rs:10]: and this

**Review by Milo123459 — CHANGES_REQUESTED** [review]

File: src/nixpacks/nix/mod.rs (hunk 1 above)
Comment (coffee-cup) [reply src/nixpacks/nix/mod.rs:9]: Yes this is a draft and not done yet

Comment (coffee-cup) [reply src/nixpacks/nix/mod.rs]: All updated now 😄

**Pull request merged**